package.json

```diff
@@ -49,6 +49,7 @@
     "@babel/preset-flow": "^7.0.0",
     "@babel/preset-react": "^7.0.0",
     "@babel/runtime": "^7.4.3",
+    "@babel/standalone": "^7.5.5",
```

Comment:
Where is this used? I can't find it anywhere

Reply:
Hmm you might be right...